src/config.go

```diff
@@ -38,13 +38,14 @@ type static struct {
 }
 
 type metric struct {
-	Name             string   `yaml:"name"`
-	Statistics       []string `yaml:"statistics"`
-	Period           int      `yaml:"period"`
-	Length           int      `yaml:"length"`
-	Delay            int      `yaml:"delay"`
-	NilToZero        bool     `yaml:"nilToZero"`
-	DisableTimestamp bool     `yaml:"disableTimestamp"`
+	Name             string      `yaml:"name"`
+	Statistics       []string    `yaml:"statistics"`
+	Dimensions       []dimension `yaml:"dimensions"`
```

Comment:
I would suggest that we rename Dimensions to additionalDimensions in config.go as the user should not be irritated by its name.